rust/cardano-blockchain-types/Cargo.toml

```
@@ -18,9 +18,7 @@ crate-type = ["cdylib", "rlib"]
 workspace = true
 
 [dependencies]
-pallas = { version = "0.33.0" }
-# pallas-hardano = { version = "0.33.0" }
-catalyst-types = { version = "0.0.5", git = "https://github.com/input-output-hk/catalyst-libs.git", tag = "catalyst-types-v0.0.5" }
+catalyst-types = { version = "0.1.5", git = "https://github.com/input-output-hk/catalyst-libs.git", branch = "wasm/catalyst-types" }
```

```suggestion
catalyst-types = { version = "0.0.6", git = "https://github.com/input-output-hk/catalyst-libs.git", tag = "catalyst-types/v0.0.6" }
```
For same reason.  Also be aware of tag name `catalyst-types-v0.0.5` does not conform to our tagging convention.